orchagent/portsorch.cpp

```diff
@@ -63,8 +63,8 @@ extern string gMyAsicName;
 #define PORT_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS     1000
 #define PORT_BUFFER_DROP_STAT_POLLING_INTERVAL_MS     60000
 #define QUEUE_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS   10000
-#define QUEUE_WATERMARK_FLEX_STAT_COUNTER_POLL_MSECS "10000"
-#define PG_WATERMARK_FLEX_STAT_COUNTER_POLL_MSECS    "10000"
+#define QUEUE_WATERMARK_FLEX_STAT_COUNTER_POLL_MSECS "60000"
+#define PG_WATERMARK_FLEX_STAT_COUNTER_POLL_MSECS    "60000"
```

Comment:
There is also a buffer pool watermark. The default needs to be changed for that as well

Reply:
There are only 3 counters for the buffer pool watermark related, I didn't change the default value since it won't make difference on the CPU consumption. 
But I think you are right, since it would employ same configuration of watermark interval, it should be same with other watermark intervals by default.